pypeit/coadd1d.py

```diff
@@ -239,13 +244,10 @@ def save(self, coaddfile, telluric=None, obj_model=None, overwrite=True):
         self.coaddfile = coaddfile
         wave_mask = self.wave_coadd > 1.0
         # Generate the DataContainer
-        onespec = OneSpec(self.wave_coadd[wave_mask],
-                          self.flux_coadd[wave_mask],
-                          PYP_SPEC=self.spectrograph.name,
-                          ivar=self.ivar_coadd[wave_mask],
+        onespec = OneSpec(self.wave_coadd[wave_mask], self.flux_coadd[wave_mask],
+                          PYP_SPEC=self.spectrograph.name, ivar=self.ivar_coadd[wave_mask],
                           mask=self.mask_coadd[wave_mask].astype(int),
-                          ext_mode=self.par['ex_value'],
-                          fluxed=self.par['flux_value'])
+                          ext_mode=self.par['ex_value'], fluxed=self.par['flux_value'])
```

Comment:
These mask variables should probably be changed to gpm to be consistent with there other changes.

Reply:
I implemented this change here, but stopped short of a wholesale replacement throughout the code I touched.